README.md

```diff
@@ -1,91 +1,90 @@
 # Sern Handler
-<a href="https://www.npmjs.com/package/sern-handler">
-<img src="https://img.shields.io/npm/v/sern_handler?maxAge=3600" alt="NPM version" /></a> <a href="https://www.npmjs.com/package/sern-handler"><img src="https://img.shields.io/npm/dt/sern_handler?maxAge=3600" alt="NPM downloads" /></a> <a href="https://www.npmjs.com/package/sern-handler"><img src="https://img.shields.io/badge/builds-stable" alt="Builds Passing"></a>
 
-Sern can automate and streamline development of your discord bot with new version compatibility and full customization.
+<a href="https://www.npmjs.com/package/@sern/handler">
+<img src="https://img.shields.io/npm/v/@sern/handler?maxAge=3600" alt="NPM version" /></a> <a href="https://www.npmjs.com/package/@sern/handler"><img src="https://img.shields.io/npm/dt/@sern/handler?maxAge=3600" alt="NPM downloads" /></a> <a href="https://www.npmjs.com/package/@sern/handler"><img src="https://img.shields.io/badge/builds-stable" alt="Builds Passing"></a>
 
--   A reincarnation of [this old project](https://github.com/jacoobes/sern_handler)
+A discord.js wrapper for you to automate and streamline your bot development.
+
+- A reincarnation of [this old project](https://github.com/jacoobes/sern_handler)
```

Comment:
The repository is now private, I suggest remove the line

Reply:
Done